### PR TITLE
Adds light order support to @airswap/types and @airswap/utils

### PR DESCRIPTION
**Pull request by dmosites**

### CODE REVIEW

File: tools/utils/src/orders.ts [hunk 1]
```diff
@@ -244,3 +247,46 @@ export function isValidOrder(order: Order): boolean {
   }
   return false
 }
+
+export function createLightOrder({
+  nonce = Date.now().toString(),
+  expiry = Math.round(Date.now() / 1000 + SECONDS_IN_DAY).toString(),
+  signerToken = ADDRESS_ZERO,
```
Comment (vdrg) [inline tools/utils/src/orders.ts:254]: Maybe we should not set default values for tokens, wallets and amounts, as the user should always provide these

**Review by ejwessel — APPROVED** [review]
besides vicente's comment this LGTM.

File: tools/utils/src/orders.ts (hunk 1 above)
Comment (dmosites) [reply tools/utils/src/orders.ts:254]: The purpose of this function is to create a complete order from partial values, so think we should keep it as-is, but could rename to `createLightOrderWithDefaults`. Thoughts?

Comment (dmosites) [reply tools/utils/src/orders.ts:254]: Going to remove this and use an object literal in the test.

**Pull request merged**